.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+source = cloudscraper
```

Comment:
Unintentional?

Reply:
Yes. Unfortunately, some of this is reused from some older work that I've done: https://github.com/VeNoMouS/cloudscraper/commit/a4dacace381713ba8137cd723b7370e7616c6ad8

I hate the association but the work is my own.